src/redux/books/books.js

```diff
@@ -0,0 +1,19 @@
+const ADD_BOOK = 'ADD_BOOK';
+const REMOVE_BOOK = 'REMOVE_BOOK';
+
+export const addBook = () => ({
+  type: ADD_BOOK,
+});
+
+export const removeBook = () => ({
+  type: REMOVE_BOOK,
+});
+
+export default function booksReducer(state = [], action) {
+  switch (action.type) {
+    case addBook.fulfilled.type:
```

Comment:
- [ ] As per this [requirement](https://github.com/microverseinc/curriculum-react-redux/blob/main/bookstore/add_reducers_an_actions.md#:~:text=Define%20state%20changes%20for%20the%20actions%20that%20you%20created.), Kindly make the function returns a new state object in which the books array will contain a new book object, passed by `action.payload`.

Reply:
done